### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by svemu**
I recently joined the GH actions team and followed the tutorial, found some super minor nitpicky inconsistencies in the instructions and hence proposing these changes

### Why:

Closes [issue link]

### What's being changed:

Proposing changes in Step 1 (Sub steps 4 and 7), Step 2( Sub step 3) 

### Check off the following:

- [ ] For workflow changes, I have verified the Actions workflows function as expected.
- [x] For content changes, I have reviewed the [style guide](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -121,7 +121,7 @@ In this step of our exercise, we will add a "build" job. We will specify `ubuntu
        name: Post welcome comment
        runs-on: ubuntu-latest
    ```
-3. Click **Start commit** in the top right of the workflow editor.
+3. Click **Commit changes** in the top right of the workflow editor.
 4. Type your commit message and commit your changes directly to your branch.
```
Comment (bpires) [inline README.md:125]: "Start commit" is actually right. This is the button we find in the top right of the workflow editor. There is no need for changes in item 3.

On the other hand, "Commit changes" is the button we will click on item 4, after typing the commit message. So, item 4 can be re-written for clarity:

```
4. Type your commit message and click **Commit changes** to commit your changes directly to your branch.
```

For quick check:

image

File: README.md [hunk 2]
```diff
@@ -70,7 +70,7 @@ To get you started, we used actions to go ahead and made a branch and pull reque
 1. Open a new browser tab, and navigate to this same repository. Then, work on the steps in your second tab while you read the instructions in this tab.
 1. Navigate to the **Code** tab.
 1. From the **main** branch dropdown, click on the **welcome-workflow** branch.
-1. Navigate to the `.github/workflows/` folder, then select **Add file** and click on **Create new file**.
+1. Navigate to the `.github/workflows/` folder, then select **Add file** and click on **+ New file**.
```
Comment (bpires) [inline README.md:73]: The correct button we would find is indeed "Create new file", not "+ New file".

In fact, the "Add file" label is hidden and pops up only on mouse hover.

IMHO, a better fix to make it clearer would be:

```
1. Navigate to the `.github/workflows/` folder, then select the "**+**" sign (Add file) on the top right and click on **Create new file**.
```

For quick check:

image

**Review by bpires — CHANGES_REQUESTED** [review]
Hey there @svemu! 👋🏻 

I'm a GitHub user and I've just done this skill course. I couldn't help to notice that you may have mixed up the buttons in the final text of your PR. Please accept these suggestions if adequate. Cheers!

**Review by svemu — COMMENTED** [review]
LGTM

**Comment by saifalsowaidi** [comment]
UTC

**Pull request closed without merging**